500 Error after latest merge

The docker log shows the following:

Error: Cannot find module 'webpack'
Require stack:
- /app/node_modules/workbox-webpack-plugin/build/generate-sw.js
- /app/node_modules/workbox-webpack-plugin/build/index.js
- /app/node_modules/next-pwa/index.js
- /app/next.config.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1077:15)
    at /app/node_modules/next/dist/server/require-hook.js:185:36
    at Module._load (node:internal/modules/cjs/loader:922:27)
    at Module.require (node:internal/modules/cjs/loader:1143:19)
    at require (node:internal/modules/cjs/helpers:110:18)
    at Object.<anonymous> (/app/node_modules/workbox-webpack-plugin/build/generate-sw.js:18:35)
    at Module._compile (node:internal/modules/cjs/loader:1256:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1310:10)
    at Module.load (node:internal/modules/cjs/loader:1119:32)
    at Module._load (node:internal/modules/cjs/loader:960:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/app/node_modules/workbox-webpack-plugin/build/generate-sw.js',
    '/app/node_modules/workbox-webpack-plugin/build/index.js',
    '/app/node_modules/next-pwa/index.js',
    '/app/next.config.js'
  ]
}

Is there a fix for the error?

--- a/package.json
+++ b/package.json
@@ -78,6 +78,7 @@
     "@types/react-syntax-highlighter": "^15.5.7",
     "prettier": "^3.0.0",
     "prettier-plugin-tailwindcss": "^0.4.0",
-    "prisma": "^4.16.2"
+    "prisma": "^4.16.2",
+    "webpack": "^5.88.1"
   }
 }

